src/main/java/org/broadinstitute/hellbender/utils/hdf5/HDF5File.java

```diff
@@ -115,6 +116,21 @@ public void close() {
     }
 
     /**
+     * Flush this file.
```

Comment:
The javadoc above calls it a file reader, not a file.  We need to be consistent.